src/scripts/utils.js

```diff
@@ -7,3 +7,13 @@
 export const getHexString = input => {
   return input.toString('hex');
 };
+
+/**
+ * Convert BTC to Satoshi.
+ *
+ * @param btc btc
+ * @returns satoshi
+ */
+export const toSatoshi = btc => {
+  return (btc * 100000000 * 10) / 10;
```

Comment:
Why `* 10) / 10`? Just multiplying the BTC amount times `100000000` should convert it to satoshis.

Reply:
Sorry didn't remove that I was testing something.